yaml/ver10/cloudintegration/yaml/cloudintegration.yaml

```diff
@@ -110,16 +95,13 @@ components:
       type: object
       properties:
         CamerasConfiguration:
```

Comment:
Guess the structure name should be singlular as we made array at higher level? 

Also Should this be "CameraConfiguration" or "DeviceConfiguration" when we are creating the component as "#/components/schemas/DeviceConfiguration"?